v20.2/create-view.md

```diff
@@ -21,6 +21,7 @@ The user must have the `CREATE` [privilege](authorization.html#assign-privileges
 Parameter | Description
 ----------|------------
 `IF NOT EXISTS` |  Create a new view only if a view of the same name does not already exist. If one does exist, do not return an error.<br><br>Note that `IF NOT EXISTS` checks the view name only. It does not check if an existing view has the same columns as the new view.
+`OR REPLACE`  |  <span class="version-tag">New in v20.2:</span> If a view of the same name already exists, replace that view.<br><br>In order to replace an existing view, the new view must have the same columns as the existing view, or more. For example, if the existing view has columns `a, b`, the new view can have an additional column `c`, but must have columns `a, b`.
```

Comment:
We also should specify that the old columns must be a prefix of the old columns. We won't allow the view to have columns (b, a) in the case here.

Reply:
Ah. Gotcha. I updated the description again.